packages/utils-react/src/key-animation/__styles__/key-animation.ts

```diff
@@ -69,7 +72,7 @@ export const keyhole = css``
 export const keyholeMask = css`
   width: 2.5rem;
   height: 3.5rem;
-  background: var(--color-dark-blue);
+  background: #262f68;
```

Comment:
some reason the var didn't work here making the mask invisible 🤷‍♀️  not sure why but this resolves the problem

Reply:
You got the variable wrong!
<img width="410" alt="Screenshot 2021-11-02 at 12 40 41" src="https://user-images.githubusercontent.com/4630495/139848266-cc0555aa-e366-43e6-a538-408f0787bb0e.png">

Reply:
Referring to your docs 😂

Reply:
🙄 typical 😂

Reply:
<img width="1260" alt="Screenshot 2021-11-02 at 13 27 07" src="https://user-images.githubusercontent.com/11574250/139855880-96c2e190-efc6-460c-be8b-97141b5e8736.png">
Still doesn't work!